CHANGES.rst

```diff
@@ -9,6 +9,8 @@
 - Turn off 2.7 Travis testing
 - Integrated arclines into PypeIt
 - Modified debug/developer modes
+- Update SpecObjs class; ndarray instead of list;  set() method
```

Comment:
I should think there were *many* more CHANGES than this.

Reply:
More changes added.